### DISCUSSION TITLE
Interactivity API

### DISCUSSION THREAD

**Discussion by JUVOJustin** [timber/timber#3038] [Ideas]
I just started looking into the activity api of wordpress. Although it is mostly used in combination with blocks it might be usable to have a timber integration for the interactivity api. Timber could be used inside the render.php and it might be especially usefull to have an integration when not working with blocks.

**Comment by Levdbas** [comment]
I read about the Interactivity API a bit and I like the concept. But I don't really see how Timber could provide a way to interact with this API in a meaningful way. Before you can use the interaction API we should first provide a groundwork for working with blocks themselves in the first place. We do have a tutorial on how to create ACF blocks, but we might want to venture further into this territory. Maybe to provide helpers to create native blocks with Timber rendering? After that, we might want to venture into the whole interactivity, but I think these are pretty much bound to personal preference.

But I would like to know how you would like to see this
